packages/next/client/image.tsx

```diff
@@ -369,6 +369,15 @@ export default function Image({
     }
   }
 
+  if (layout === 'intrinsic' || layout === 'fixed') {
```

Comment:
I would move this assignment up to line 296 where `imgStyle` is declared because theres no reason why it needs to be down here (both heightInt and priority are defined before line 296 already). Then you dont need to shallow clone the entire `imgStyle` object.

Reply:
Done.